modules/home/my_home/default.nix

```diff
@@ -1,17 +1,16 @@
 { options, config, pkgs, lib, inputs, ... }:
 let
   cfg = config.default-home;
+
+  user = config.snowfallorg.user;
```

Comment:
I realize this wasn't documented, but there is now a set of modules that are added for extra utility! Right now they provide:

NixOS / Darwin:

`config.snowfallorg.user.<name>.create` (default `true`)
Whether to create the user account automatically.

`config.snowfallorg.user.<name>.home.path` (default is the inferred home based on system and user name)
The home path for the user account.

`config.snowfallorg.user.<name>.home.config` (default `{}`)
Configuration passed to `home-manager.users.<name>`.

Home Manager:

`config.snowfallorg.user.enable` (default `true`)
Whether to configure the user.

`config.snowfallorg.user.name` (default `""`)
The user name.

`config.snowfallorg.user.home.directory` (default is the inferred home based on system and user name)
The user's home directory.

---

What this means is that if you want to set home-manager configuration from within your NixOS or nix-darwin configuration, you can use `config.snowfallorg.user.arichtman.home.config` and it will be proxied to `home-manager.users.arichtman`. This is done because `home-manager.users.<name>` is being set to a function in order to get home-manager specific arguments.